pkg/runtime/config.go

```diff
@@ -40,4 +41,10 @@ func (c *Config) BindFlags() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.",
 	)
+	flag.BoolVar(
+		&c.EnableDevModeLogging, "enable-dev-mode-logging",
+		false,
+		"Configures the logger to use a Zap development config (stacktraces on warnings, no sampling), "+
```

Comment:
Nit: Probably, we can add more info about logLevel as well, something like this: `Development Mode, defaults(encoder=consoleEncoder,logLevel=Debug,stackTraceLevel=Warn, no sampling) + "Production Mode defaults(encoder=jsonEncoder,logLevel=Info,stackTraceLevel=Error), sampling")`

Reply:
sure. looks better